includes/DefaultSettings.php

```diff
@@ -512,7 +526,7 @@
  *
  * Please specify the namespace, as in the example below.
  */
-$wgRepositoryBaseUrl = "https://commons.wikimedia.org/wiki/File:";
+$wgRepositoryBaseUrl = "http://commons.wikimedia.org/wiki/File:";
```

Comment:
wikimedia.org switched to HTTPS some time ago. Let's keep using HTTPS here.

Reply:
I kept 1.19.24 `DefaultSettings.php` untouched and added an override to `includes/wikia/DefaultSettings.php` https://github.com/Wikia/app/pull/14597/files#diff-013f1f772182745f48c5f1429d0cdd35R27